kubernetes-extension-balo/src/main/ballerina/ballerinax/src/knative/annotation.bal

```diff
@@ -0,0 +1,269 @@
+// Copyright (c) 2019 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
We need to move this annotations to ballerina lang repo. Example: https://github.com/ballerina-platform/ballerina-lang/tree/master/stdlib/kubernetes